src/main/java/edu/kis/powp/command/ComplexCommand.java

```diff
@@ -0,0 +1,22 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+    private ArrayList<ComplexCommandStore> commands = new ArrayList<>();
```

Comment:
List should hold other DriverCommands (Composite pattern)
List type declaration would be sufficient and would not limit the implementation.